script.js

```diff
@@ -1,12 +1,64 @@
+/* Homepage Functions */
+function on() {
+  document.getElementById("overlay").style.display = "block";
+}
 
+function off() {
+  document.getElementById("overlay").style.display = "none";
+}
 
-function on() {
-    document.getElementById("overlay").style.display = "block";
-  }
-  
-  function off() {
-    document.getElementById("overlay").style.display = "none";
-  }
+/* Dynamic Project Functions */
+function projectOn() {
+  document.getElementById("output").style.display = "block";
+}
+
+function projectOff() {
+  document.getElementById("output").style.display = "none";
+}
 
+
+
+function getProject(a) {
+  projectOn();
+fetch("projects.json")
+.then((res) => res.json())
+.then((data) => {
+ output = `
+ <div class="project_overlay">
+  <div class="project_structure_overlay">
+      <div> 
+          <button onclick="projectOff()" class="project_overlay_btn_close"> X </button>
+      </div>
+      <div class="project_heading_overlay">
+              <h2 class="title_overlay"> ${data[a].name} </h2>
+              <ul class="project_body_title_list">
+                  <li class="title_list_main_overlay">${data[a].projectClient}</li>
+                  <li><img class="counter_overlay" src="./Assets/Counter.png" >${data[a].projectInfo[0]}</li>
+                  <li><img class="counter_overlay" src="./Assets/Counter.png" >${data[a].projectInfo[1]}</li>
+              </ul>
+      </div>
+      <div> <img class="image_ovlerlay" src=${data[a].image} alt="snapshoot"> </div>
+      <div class="project_body_overlay">
+          <div class="text_group_overlay">
+              <p>${data[a].mainText}</p>
+          </div>
+          <div class="btn_group_overlay">
+          <ul class="project_tech_overlay">
+                  <li>${data[a].technologies[0]}</li>
+                  <li>${data[a].technologies[1]}</li>
+                  <li>${data[a].technologies[2]}</li>
+              </ul>
+              <hr class="hr_style">
```

Comment:
- [ ] I think here there is no need to add hr line since it is not found in the figma template so kindly remove it 👍🏻♻🚀